bin/public/js/term.js

```diff
@@ -1328,7 +1328,8 @@ Terminal.prototype.refresh = function(start, end) {
                 out += 'font-weight:bold;';
               }
               // See: XTerm*boldColors
-              if (fg < 8) fg += 8;
+              if(this.options.boldAsBright > 0 && fg < 8)
```

Comment:
Can you submit this change to `term.js` as well? Otherwise when updating the `term.js` version, this will be overridden.